src/main/java/com/winner_is_kungen/tda367/view/canvas/InfiniteCanvasBlock.java

```diff
@@ -125,7 +120,7 @@ public final void setContent(Node value) {
 		}
 	}
 
-	public final ObjectProperty<Node> contentProperty() {
+	 final ObjectProperty<Node> contentProperty() {

```

Comment:
```suggestion
	final ObjectProperty<Node> contentProperty() {
```

One space too much. Someone didn't run the code formatter.